sdks/java/core/src/main/java/org/apache/beam/sdk/Pipeline.java

```diff
@@ -125,9 +126,20 @@
     /**
      * Wraps {@code cause} into a {@link PipelineExecutionException}.
      */
+    public static PipelineExecutionException wrap(String transformName, Throwable cause) {
+      if (Strings.isNullOrEmpty(transformName)) {
```

Comment:
`transformName` should be explicitly `@Nullable` and we should check only vs `null`. Conflating `null` and `""` is strongly discouraged by the Guava authors when they talk about this exact case.

Reply:
Done